readme.md

```diff
@@ -105,7 +116,10 @@ Your Answer:
 Write some basic CSS that demonstrates changing a CSS property when the device width drops below 40rem.
 
 ```css
-
+.device {
+  width: 40em;
+  font-size: 12em;
+}
 ```
```

Comment:
You need something here that will trigger this `.device` selector if the device width drops below 40rem. Right now this selector is being applied to anything with a class of `device`, regardless of device width. It should look something like this...

```js
@media (max-width: 40rem) {
  body{
    background: lemonchiffon;
  }
}
```